kotlin/app/src/main/res/layout/activity_checkout.xml

```diff
@@ -77,20 +77,19 @@
             layout="@layout/buy_with_googlepay_button"
             android:layout_width="match_parent"
             android:layout_height="@dimen/buy_button_height"
-            android:layout_marginBottom="@dimen/buy_button_margin_bottom"
             android:visibility="gone" />
 
-        <com.google.android.material.button.MaterialButton
-            xmlns:app="http://schemas.android.com/apk/res-auto"
+        <Button
             android:id="@+id/paymentCardOcrButton"
-            app:materialThemeOverlay="@style/Theme.MaterialComponents"
```

Comment:
Only added this because I thought it's part of the Google style guidelines. LGTM.